muirurikin/Politico
# #163939969 implement pep8 linting

**What Does This PR Do**
Modifies code to meet pep8 standards.

**Task Completed**
Follow pep8 standards while writing all you code.

**How To Test**
- Clone Repo locally.

- Cd into project folder and open the project using your editor.

- Install ```pep8``` linter on your editor

- Check files to see if there are errors

**Relevant PT Stories**
[#163939969](https://www.pivotaltracker.com/n/projects/2241669/stories/163939969)
